jut-server.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-DIR="$( cd "$( dirname "$0" )" && pwd )"
-CMD="$DIR/target/appassembler/bin/jut-server"
+CMD="target/appassembler/bin/jut-server"
```

Comment:
Sorry, seems that this commit did go into the PR by mistake. I"ll clean up the PR. 

(I got a fresh CentOS 7 installation from my department where I am setting up the ATH. Seems that the installed bash 4.2.46(1)-release (x86_64-redhat-linux-gnu) did not accept the parameters)